ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ParticipantEditPage.java

```diff
@@ -22,7 +22,8 @@
     private Map<String, String> mapLangPos = new HashMap<String, String>();
 
     // Object initialization for log
-    //private static Logger log = Logger.getLogger(ParticipantEditPage.class.getName());
+    // private static Logger log =
+    // Logger.getLogger(ParticipantEditPage.class.getName());
```

Comment:
@pmuchowski  I am surprise with the changes, I have added all the fixes, thanks for your input.